pkg/converter/convert.go

```diff
@@ -51,10 +51,12 @@ const (
 	flagAuthorityHost      = "authority-host"
 	flagFederatedTokenFile = "federated-token-file"
 	flagTokenCacheDir      = "token-cache-dir"
+	flagExecInstallHint    = "install-hint"
 
 	execName        = "kubelogin"
 	getTokenCommand = "get-token"
 	execAPIVersion  = "client.authentication.k8s.io/v1beta1"
+	execInstallHint = "Install kubelogin for use with kubectl by running az aks install-cli"
```

Comment:
```suggestion
	execInstallHint = "Install kubelogin to connect to Azure AD enabled Kubernetes clusters. Go to https://azure.github.io/kubelogin for details"
```

Reply:
i tend not to put aks context here as this tool can be used for any Azure AD k8s, even on AWS. :)

Reply:
Makes sense and thanks for that feedback. I did not commit your suggestion as is, but instead aligned with the install hint we are adding to AKS. Let me know if you didn't want that.